src/components/FeatureViews/Dashboard/Today.js

```diff
@@ -0,0 +1,95 @@
+import React, {useEffect} from 'react';
+import {useSelector} from "react-redux";
+import {useSearchSession} from "actions/searchActions";
+import * as hooks from "actions/hooks";
+import Typography from "@material-ui/core/Typography";
+import TodayCard from "./TodayCard";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import Loading from "components/Loading";
+import './Dashboard.scss';
+import Grid from '@material-ui/core/Grid';
+import gql from "graphql-tag";
+import { useQuery } from "@apollo/react-hooks";
+
+
+const Today = (filter) => {
+    const TODAY_SESSION_QUERY= gql`
+            query todaySessionQuery($filter: String="") {
+                sessionSearch(query: $filter, time: "", sort: "timeAsc") {
```

Comment:
Did you mean to query for only sessions from `today`?

Reply:
yes - @gglinoga this should only before today. otherwise you're going to get all the sessions that aren't today. this is the dashboard.